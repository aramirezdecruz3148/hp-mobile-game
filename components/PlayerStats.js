import React from "react";
import { View, Text } from "react-native";
import PropTypes from "prop-types";

export default function PlayerStats({ health }) {
  return (
  <View>
    <Text>Your Health: {health}</Text>
  </View>
  );
}

PlayerStats.propTypes = {
  health: PropTypes.number.isRequired
};