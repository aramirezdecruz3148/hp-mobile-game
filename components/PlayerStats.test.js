import React from 'react';
import { shallow } from 'enzyme';
import PlayerStats from './PlayerStats';

describe('PlayerStats component', () => {
  it('renders PlayerStats', () => {
    const wrapper = shallow(<PlayerStats health={8} />);
    expect(wrapper).toMatchSnapshot();
  });
});