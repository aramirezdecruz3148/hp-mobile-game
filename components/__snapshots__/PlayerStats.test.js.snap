// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerStats component renders PlayerStats 1`] = `
<View>
  <Text>
    Your Health: 
    8
  </Text>
</View>
`;
